cvcl/models.py

```diff
@@ -284,3 +284,58 @@ class Meta:
 
     def __str__(self):
         return str(self.id)
+
+
+class InstallPackage(models.Model):
+    name = models.CharField(max_length=30)
+    package_update = models.BooleanField(default=False)
+    package_upgrade = models.BooleanField(default=False)
+    package_reboot_if_required = models.BooleanField(default=False)
+    vm_definition = models.ForeignKey('VmDefinition', on_delete=models.CASCADE, null=True,
+                                      related_name='install_packages')
+
+    class Meta:
+        verbose_name = "Install Packages"
+
+    def __str__(self):
+        return str(self.id)
+
+
+class AddUser(models.Model):
+    name = models.CharField(max_length=30)
+    primary_group = models.CharField(max_length=50, blank=True)
+    groups = models.CharField(max_length=50, blank=True)
+    lock_passwd = models.BooleanField(default=False)
+    passwd = models.CharField(max_length=255, blank=True, null=True)
+    vm_definition = models.ForeignKey('VmDefinition', on_delete=models.CASCADE, null=True, related_name='add_users')
+
+    class Meta:
+        verbose_name = "Add Users"
+
+    def __str__(self):
+        return str(self.id)
+
+
+class AddGroup(models.Model):
```

Comment:
Why is the model called `AddGroup`? A group is a group, so it could be `UserGroup`. Don't put verbs in model names.